site/documentation/how-do-i-write-policies/index.md

```diff
@@ -786,6 +786,34 @@ undefined
 
 In some cases, having an undefined result for a document is not desirable. In those cases, policies can use the [Default Keyword](#default) to provide a fallback value.
 
+## <a name="functions"></a> Functions
+
+Rego supports user defined functions. Functions are only accessible within the modules they are defined in. Once defined, a user function behaves like a builtin.
```

Comment:
sounds good, one thing I chose is to not allow functions to use the `with` keyword, did you have thoughts on that? It seems like it would make more sense for someone to just pass an argument to the function instead